docs/content/integrations/dbt.mdx

```diff
@@ -1,380 +1,159 @@
-    context.resources.dbt.run(models=["tag:staging"])
-
-@job(resource_defs={"dbt_sync": my_remote_sync_rpc})
-def my_dbt_job():
-    run_staging_models_and_wait()
-```
-
-## Advanced Configuration
-
-For full documentation on all available config, [visit the API docs for dagster-dbt](/\_apidocs/libraries/dagster_dbt).
-
-**Generating Asset Materializations when you run your dbt project**
-
-`dagster_dbt` is configured to automatically create [Asset Materializations](/concepts/assets/asset-materializations) for each of your dbt models when you run the `run` command (via either the CLI or over RPC). These materializations are populated with metadata that is automatically parsed from the dbt response. The available metadata differs between dbt versions. If there's anything you would like to add to these materializations, you can explicitly invoke the underlying <PyObject module='dagster_dbt.utils' object='generate_materializations'/> utility function, and modify each of the resulting materializations like so:
-
-```python file=/integrations/dbt.py startafter=start_marker_dbt_asset_mats endbefore=end_marker_dbt_asset_mats dedent=4
-from dagster_dbt.utils import generate_materializations
-
-from dagster import op
-
-@op(required_resource_keys={"dbt"})
-def dbt_run_with_custom_assets(context):
-    dbt_result = context.resources.dbt.run()
-    for materialization in generate_materializations(dbt_result):
-        context.log_event(
-            materialization._replace(
-                metadata_entries=[...]  # insert whatever metadata you want here
-            )
+@repository
+def my_repo():
+    return [
+        dbt_assets,
+        ScheduleDefinition(
+            job=run_something,
+            cron_schedule="@daily",
+        ),
+        ScheduleDefinition(
+            job=run_everything,
+            cron_schedule="@weekly",
         )
-    return dbt_result
+    ]
 ```
 
-**dbt CLI: Set the dbt profile and target to load**
+See (docs for running stuff on a schedule) to learn more.
 
-```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_config_profile_and_target endbefore=end_marker_dbt_cli_config_profile_and_target dedent=4
-from dagster_dbt import dbt_cli_resource
-
-from dagster import job
+## Defining Downstream Dependencies
 
-config = {"profile": PROFILE_NAME, "target": TARGET_NAME}
+Dagster allows you to define assets that are downstream of specific dbt models. One property of dbt based assets is that the external tool (in this case, dbt), handles the loading/storing of those models internally, rather than Dagster directly storing/loading the tables that are updated.
 
-@job(resource_defs={"dbt": dbt_cli_resource.configured(config)})
-def my_job():
-    # ...
-```
+This means that there's a range of ways that we might want to load a dbt model as input to a python function. For example, you might want to load the contents of the table as a Pandas dataframe, or you might want to load the contents into a PySpark session.
```

Comment:
```suggestion
This means that there's a range of ways to load a dbt model as input to a Python function. For example, you might want to load the contents as a Pandas dataframe or into a PySpark session.
```

I wanted to tighten this up a bit. I'm not sure if my removing `... of the table` changes the meaning or accuracy of this sentence, but given there wasn't a mention of `tables` before, I thought it was okay. Feel free to put it back/tinker if I'm wrong.